operators/pkg/forge/containers_test.go

```diff
@@ -64,12 +64,9 @@ var _ = Describe("Containers and Deployment spec forging", func() {
 		portNum              = 1234
 		httpPath             = "/some/path"
 		httpPathAlternative  = "/some/different/path"
-	)
-
-	// test aware constants (are tested against code)
-	const (
-		myDriveMountPath = "/mydrive"
-		myDriveName      = "mydrive"
+		nfsServerName        = "nfs-server-name"
+		nfsPath              = "/nfs/path"
+		nfsVolumeName        = "nfs-vol"
```

Comment:
This constant is unused, and should be dropped.

Reply:
Removed